elide-contrib/elide-dynamic-config-helpers/pom.xml

```diff
@@ -42,20 +42,26 @@
         <maven.compiler.source>1.8</maven.compiler.source>
         <maven.compiler.target>1.8</maven.compiler.target>
         <hjson.version>3.0.0</hjson.version>
-        <everit.version>1.0.0</everit.version>
         <jackson.version>2.10.2</jackson.version>
         <commons-lang3.version>3.0</commons-lang3.version>
+        <commons-io.version>2.6</commons-io.version>
         <slf4j-api.version>1.7.26</slf4j-api.version>
-        <jsonschema2pojo-core.version>1.0.2</jsonschema2pojo-core.version>
         <lombok.version>1.18.10</lombok.version>
+        <handlebars.version>4.2.0</handlebars.version>
         <json-schema-validator.version>2.2.12</json-schema-validator.version>
+        <junit.jupiter.version>5.6.0</junit.jupiter.version>
```

Comment:
We should inherit the JUnit versions from the parent pom (Junit should be the same across all maven modules).

Reply:
Junit Platform is only added in dynamic config so that version is included here.

Reply:
We can move platform to the parent pom.